Felienne/hedy
# Implemented public profiles

**Description**
In this PR we introduce public profiles! Users are able to create a public profile in their profile settings. They are able to select a profile image, set a personal text and select a favorite saved program. Other users are able to visit public profiles by navigating to `/user/<username>`. When no public profile is set a 404 page is given.

**Fix for**
In this PR we fix #1595.

**How to test**
Navigate to profile settings and notice a new header: public profile. In this section you are able to set your personal public profile. Navigate to the profile by going to `/user/<username` and notice that your set values are shown.

**This is currently a draft! Already a PR to solve merge conflicts along the way**
